### PR TITLE
squid: S1213 Members of an interface declaration or class should appe…

### PR DESCRIPTION
**Pull request by devFozgul**
This Pull request is focused on  resolving occurrences of squid : S1213
You can find more information about it here: https://dev.eclipse.org/sonar/rules/show/squid:S1213
Please let me know if you have any questions.
Fevzi Ozgul

### CODE REVIEW

File: tcslackbuildnotifier-core/src/main/java/slacknotifications/teamcity/payload/content/Commit.java [hunk 1]
```diff
@@ -6,6 +6,12 @@
  * Created by Peter on 4/06/2014.
  */
 public class Commit {
+	
+	
+	private String description;
```
Comment (PeteGoo) [inline tcslackbuildnotifier-core/src/main/java/slacknotifications/teamcity/payload/content/Commit.java]: tabs and spaces seem to be messed up here

File: tcslackbuildnotifier-core/src/main/java/slacknotifications/Field.java [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 public class Field {
 
+	private String title;
+	private String value;
+	private Boolean isShort;
```
Comment (PeteGoo) [inline tcslackbuildnotifier-core/src/main/java/slacknotifications/Field.java]: more tabs

**Comment by PeteGoo** [comment]
@devFozgul can you review my comments please?

**Comment by devFozgul** [comment]
I will do it  asap.

**Comment by devFozgul** [comment]
Hi Peter,  I wonder if you could check decide on my PRs within this week?
It is important for my records that I get all my pending PRs settled within
this week, as my company DevFactory is ending this Open Source Projects
Optimization and Cleaning work by next week.  Thank you very much for your
cooperation.
Fevzi Ozgul

**Pull request closed without merging**